src/fparser/two/utils.py

```diff
@@ -629,11 +696,33 @@ def match(startcls, subclasses, endcls, reader,
                             'expected <%s-name> is %s but got %s. Ignoring.'
                             % (end_stmt.get_type().lower(),
                                start_stmt.get_name(), end_stmt.get_name()))
-        return content,
+        return (content,)
 
     def init(self, content):
+        '''
+        Initialise the `content` attribute with the list of child nodes.
+
+        :param content: list of nodes that are children of this one.
+        :type content: list of :py:class:`fparser.two.utils.Base` or NoneType
+
+        '''
         self.content = content
-        return
+
+    def get_child(self, node_type):
```

Comment:
I think this can just be removed as we have a generic `get_child`?

Reply:
Done.